src/main/webapp/app/course/manage/overview/course-management-overview-statistics-dto.model.ts

```diff
@@ -0,0 +1,9 @@
+import { CourseManagementOverviewExerciseStatisticsDTO } from 'app/course/manage/overview/course-management-overview-exercise-statistics-dto.model';
+
+export class CourseManagementOverviewStatisticsDto {
+    public courseId: number;
+    public exerciseDTOS: CourseManagementOverviewExerciseStatisticsDTO[];
+    public activeStudents: number[];
```

Comment:
Should all these variables have a `?` at the end since they can technically be undefined?

Reply:
The server always sends values for those, so I don't think they can be undefined?

Reply:
Okay I was just wondering because of this here: `course-management-overview-exercise-details-dto.model.ts` you added question marks